projects/ngx-translate/src/lib/translate.service.ts

```diff
@@ -341,9 +349,22 @@ export class TranslateService implements ITranslateService {
     ): Observable<InterpolatableTranslationObject> {
         this.pending = true;
 
-        const loadingTranslations = this.currentLoader
-            .getTranslation(lang)
-            .pipe(shareReplay(1), take(1));
+        const loaders = Array.from(this.store.loaders.values());
```

Comment:
this should just be `this.store.getLoaders()`
I don't want the service to know about indices and implementation details in the store.